lupin/schema.py

```diff
@@ -110,9 +110,11 @@ def dump(self, obj, mapper):
         Returns:
             dict
         """
-        return {key: field.extract_attr(obj, mapper, key)
-                for key, field
-                in self._fields.items() if not field.is_write_only}
+        dict_to_filter = {key: (field.extract_attr(obj, mapper, key), field.is_ignore_if_null)
```

Comment:
je pense que c'est le bon moment pour passer sur une boucle for classique ça rendra le truc un peu plus lisible

Reply:
je suis deçu, vu le temps que jem e suis pris la tête :smiley: